crates/transaction-pool/src/pool/best.rs

```diff
@@ -61,6 +62,14 @@ pub(crate) struct BestTransactions<T: TransactionOrdering> {
     pub(crate) independent: BTreeSet<PendingTransaction<T>>,
     /// There might be the case where a yielded transactions is invalid, this will track it.
     pub(crate) invalid: HashSet<TxHash>,
+    /// Used to recieve any new pending transactions that have been added to the pool after this
+    /// iterator was snapshotted
+    ///
+    /// We prioritize these new transactions over existing ones by first inserting these into the
```

Comment:
I think prioritize is not correct here, since they get the same treatment as txs already included in this snapshot, it's just an extra data source.

Reply:
ah yeah given what priority actually means in the pool data structures - prioritize is the wrong word here. 
makes sense, i removed this comment completely.